### ISSUE TITLE
Install tool:  fatal error after database connection setup

### ISSUE THREAD

**Issue by importer** [contao/core-bundle#80]
> <a href="https://github.com/xchs"><img src="https://avatars.githubusercontent.com/u/1218766?v=3" align="left" width="42" height="42" hspace="10"></img></a> [Issue](https://github.com/contao/contao/issues/2) by @xchs
Tuesday Jun 10, 2014 at 21:04 GMT

The Contao 4.0.0-alpha1 install tool throws a fatal error when it comes to the database connection setup:

```
Fatal error: Uncaught exception Exception with message Query error: Table 'contao4.tl_user' doesn't exist (SELECT COUNT(*) AS count FROM tl_user WHERE admin=1) thrown in vendor/contao/contao-library/src/Contao/Database/Statement.php on line 283

#0 vendor/contao/contao-library/src/Contao/Database/Statement.php(254): Contao\Database\Statement->query()
#1 vendor/contao/contao-library/src/Contao/Database.php(190): Contao\Database\Statement->execute()
#2 vendor/contao/module-core/src/controllers/BackendInstall.php(577): Contao\Database->execute('SELECT COUNT(*)...')
#3 vendor/contao/module-core/src/controllers/BackendInstall.php(191): Contao\BackendInstall->createAdminUser()
#4 web/contao/install.php(26): Contao\BackendInstall->run()
#5 {main}
```

**Comment by importer** [comment]
> <a href="https://github.com/leofeyer"><img src="https://avatars.githubusercontent.com/u/1192057?v=3" align="left" width="42" height="42" hspace="10"></img></a> Comment by @leofeyer
Wednesday Jun 11, 2014 at 14:22 GMT

Fixed in contao/module-core@9294261b09e020600bb955a54f61dd431637cbb6.

**Comment by importer** [comment]
> <a href="https://github.com/xchs"><img src="https://avatars.githubusercontent.com/u/1218766?v=3" align="left" width="42" height="42" hspace="10"></img></a> Comment by @xchs
Wednesday Jun 11, 2014 at 20:24 GMT

Thanks a lot.
